x-pack/plugins/ml/public/application/services/results_service/result_service_rx.ts

```diff
@@ -68,8 +71,12 @@ export function resultsServiceRxProvider(mlApiServices: MlApiServices) {
       timeFieldName: string,
       earliestMs: number,
       latestMs: number,
-      intervalMs: number
+      intervalMs: number,
+      dataFeedConfig?: DatafeedOverride
     ): Observable<MetricData> {
+      const scriptFields: any | undefined = dataFeedConfig?.script_fields;
+      const aggFields: Aggregation | undefined = dataFeedConfig?.aggregations;
```

Comment:
Are the `: any | undefined / : Aggregation | undefined` parts necessary, I assumed they should be inferred on assignment?

Reply:
Removed here https://github.com/elastic/kibana/pull/81923/commits/d96a119950e710b043eb3c89568be037cc203ed2